docs/src/using.md

```diff
@@ -69,7 +70,12 @@ curl localhost:3000/catalog | jq
       "end": 65533
     },
     ...
-  }
+  },
+  "styles": {
+    "americana": {
+      "path": "path/to/americana.json",
```

Comment:
Note: schema has to be able to fit rendering.
I think current schema can, but please double check my reasoning here.
Depending on how we want to configure this, different catalog design might be nessesary.

I am thinking about parameters like:
- allow rendering file format xyz
- allow rendering

=> things which likely will have to be configured at the styles level.

Unsure if more options are nessesary to show..